src/context-selection/use-context-selection/use-context-selection.js

```diff
@@ -27,6 +27,14 @@ export function usePeriodId() {
     return useQueryParam('periodId', PARAMS_SCHEMA.periodId)
 }
 
+/**
+ *
+ * @returns the selected attribute-options in the shape of
+ * ```{
+ *   categoryId: categoryOptionId
+ * }```
```

Comment:
```suggestion
 * ```{
 *   [categoryId: String]: String // categoryOptionId
 * }```
```